src/ajax/map_variants.php

```diff
@@ -198,6 +198,195 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
 
 
 
+
+// Performing lift-overs so variants are described as relative to as many GBs as possible.
+$aActiveGBs = $_DB->query('
+        SELECT id,
+            CONCAT("VariantOnGenome/DNA", if(column_suffix = "", "", "/"), column_suffix) as "DNA",
+            CONCAT("position_g_start", if(column_suffix = "", "", "_"), column_suffix) as "position_start",
+            CONCAT("position_g_end", if(column_suffix = "", "", "_"), column_suffix) as "position_end"
+        FROM ' . TABLE_GENOME_BUILDS
+)->fetchAllGroupAssoc();
+
+if (count($aActiveGBs) < 1) {
```

Comment:
This `if()` can never be true. How has the code been tested? How many builds were active?

Reply:
Oops ':)